static/app/components/searchQueryBuilder/tokens/filter/utils.tsx

```diff
@@ -128,14 +128,20 @@ export function unescapeTagValue(value: string): string {
   return value.replace(/\\"/g, '"');
 }
 
-export function formatFilterValue(token: TokenResult<Token.FILTER>['value']): string {
+export function formatFilterValue(
+  token: TokenResult<Token.FILTER>['value'],
+  allContains = false
```

Comment:
Instead of adding a second argument, can you make this `{token, allContains}`?

Reply:
Also, I wonder if the naming can be a bit more descriptive for what it is doing in this function. Like `stripWildcards` or something

Reply:
Hrrm, this function does other things than just stripping wildcards, such as formatting for date values 🤔

Reply:
Ahhhh sorry 🤦